packages/dotnet-sdk/src/TeamsFx/Bot/TeamsBotSsoPromptSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Microsoft.TeamsFx;
+
+/// <summary>
+/// Contains settings for an <see cref="TeamsBotSsoPrompt"/>.
+/// </summary>
+public class TeamsBotSsoPromptSettings
+{
+    /// <summary>
+    /// Gets or sets the array of strings that declare the desired permissions and the resources requested.
+    /// </summary>
+    /// <value>The array of strings that declare the desired permissions and the resources requested.</value>
+    public string[] Scopes { get; set; }
+
+    /// <summary>
+    /// Gets or sets the number of milliseconds the prompt waits for the user to authenticate.
+    /// Default is 900,000 (15 minutes).
+    /// </summary>
+    /// <value>The number of milliseconds the prompt waits for the user to authenticate.</value>
+    public int? Timeout { get; set; }
```

Comment:
should we set a default value to here directly instead of setting the default value in another place

Reply:
Fixed.